client/app/src/Controller/Ndr/IncomeBenefitController.php

```diff
@@ -38,39 +39,47 @@ class IncomeBenefitController extends AbstractController
      * @var StepRedirector
      */
     private $stepRedirector;
+    private $clientApi;
 
     public function __construct(
         ReportApi $reportApi,
         RestClient $restClient,
-        StepRedirector $stepRedirector
+        StepRedirector $stepRedirector,
+        ClientApi $clientApi
     ) {
         $this->reportApi = $reportApi;
         $this->restClient = $restClient;
         $this->stepRedirector = $stepRedirector;
+        $this->clientApi = $clientApi;
     }
 
     /**
      * @Route("/ndr/{ndrId}/income-benefits", name="ndr_income_benefits")
-     * @Template("@App/Ndr/IncomeBenefit/start.html.twig")
      *
-     * @param $ndrId
+     * @Template("@App/Ndr/IncomeBenefit/start.html.twig")
      *
      * @return array|RedirectResponse
      */
     public function startAction($ndrId)
     {
+        /** @var User $user */
+        $user = $this->getUser();
+        $isMultiClientDeputy = 'ROLE_LAY_DEPUTY' == $user->getRoleName() ? $this->clientApi->checkDeputyHasMultiClients($user->getDeputyUid()) : null;
```

Comment:
I thought there might have been a simpler way for this, but it would require refactoring `checkDeputyHasMultiClients` as this function is only for lay's we should have done the checking if they are a lay inside the function rather than have a ternary check. 

With the above in mind, I will suggest changing the null to a false just for type consistency.